qiita_db/handlers/artifact.py

```diff
@@ -315,9 +315,9 @@ def post(self):
         values['template'] = prep_id
         cmd = qdb.software.Command.get_validator(atype)
         params = qdb.software.Parameters.load(cmd, values_dict=values)
-        if add_default_workflow:
+        if add_default_workflow or add_default_workflow == 'True':
```

Comment:
just to confirm, we expect add_default_workflow to be the string 'True' in that exact case? It seems like it should be add_default_workflow is True instead. Or add_default_workflow.lower() == 'true'.

Reply:
We expect to either be True (python boolean) or it's text: 'True'

Reply:
gotcha!